packages/database/src/migrations/20210112015125-AddHeatmapContactsConfirmedSuspectedCasesSCovidSamoa-modifies-data.js

```diff
@@ -0,0 +1,86 @@
+'use strict';
+
+import { insertObject, generateId, codeToId } from '../utilities';
+
+var dbm;
+var type;
+var seed;
+
+/**
+ * We receive the dbmigrate dependency from dbmigrate initially.
+ * This enables us to not have to rely on NODE_PATH.
+ */
+exports.setup = function (options, seedLink) {
+  dbm = options.dbmigrate;
+  type = dbm.dataType;
+  seed = seedLink;
+};
+
+async function deleteMapOverlay(db, mapOverlayId) {
+  return db.runSql(`
+    DELETE FROM
+      "mapOverlay"
+    WHERE
+      "id" = '${mapOverlayId}';
+  `);
+}
+
+async function deleteMapOverlayInMapOverlayGroupRelation(db, childId) {
+  return db.runSql(`
+    DELETE FROM
+      "map_overlay_group_relation"
+    WHERE
+      "child_id" = '${childId}';
+  `);
+}
+
+const mapOverlay = {
+  id: 'COVID_WS_Address_Contacts_Of_Suspected_And_Confirmed_Cases_WS',
+  name: 'Home village of contacts of suspected/confirmed cases',
+  userGroup: 'Donor',
+  dataElementCode: 'value',
+  measureBuilderConfig: {
+    dataValues: {
+      WS_CD_CT_Contact_12: '*',
+      WS_CD_CT_Contact_06: '1. Visit 1',
+    },
+    programCode: 'SC1CT',
+    dataSourceType: 'custom',
+    entityAggregation: {
+      aggregationType: 'REPLACE_ORG_UNIT_WITH_ORG_GROUP',
+      dataSourceEntityType: 'case',
+      aggregationEntityType: 'village',
+    },
+  },
+  measureBuilder: 'countEventsPerOrgUnit',
+  presentationOptions: {
+    scaleType: 'neutral',
+    displayType: 'spectrum',
+    measureLevel: 'Village',
+    hideByDefault: {
+      null: true,
+    },
+  },
+  countryCodes: '{WS}',
+  projectCodes: '{covid_samoa}',
+};
+
+exports.up = async function (db) {
+  await insertObject(db, 'mapOverlay', mapOverlay);
+  const MapOverLayGroupId = await codeToId(db, 'map_overlay_group', 'COVID19_Samoa');
```

Comment:
* We use `camelCase` for object instances and primitive values
* We use `PascalCase` for class names

```suggestion
  const mapOverLayGroupId = await codeToId(db, 'map_overlay_group', 'COVID19_Samoa');
```

In case you use VSCode as your editor, you can quickly and safely rename a variable using `F2` (default shortcut )